### ISSUE TITLE
Support raxml-AVX2 if available

### ISSUE THREAD

**Issue by tseemann** [nickjcroucher/gubbins#196]
```
class RAxMLExecutable(object):
	def __init__(self, threads, model = 'GTRCAT', verbose = False ):
		self.verbose = verbose
		self.threads = threads
		self.single_threaded_executables = ['raxmlHPC-AVX','raxmlHPC-SSE3','raxmlHPC']
		self.multi_threaded_executables = ['raxmlHPC-PTHREADS-AVX','raxmlHPC-PTHREADS-SSE3','raxmlHPC-PTHREADS']
		self.model = model
```

Can you add `'raxmlHPC-PTHREADS-AVX2` to the start of the list?

It will be installed by Brew if your CPU supports it and you build from source.

But it may not exist for some people (I assume you fall back until one exists?

**Comment by andrewjpage** [comment]
Fixed in v2.2.2 (for which there is a release)

**Comment by tseemann** [comment]
Thanks @andrewjpage !
